docs/events.md

```diff
@@ -597,25 +503,42 @@ Event that represents a thing's config was updated.
   JSON in the following format:
 
   - `id` **String** thing's ID
-  - `config` **Array** list of updated config
+  - `config` **Array** - **Optional** config items, each one formed by:
     - `sensorId` **Number** sensor ID
-    - `change` **Boolean** enable sending sensor data when its value changes
-    - `timeSec` **Number** - **Optional** time interval in seconds that indicates when data must be sent to the cloud
-    - `lowerThreshold` **Optional (Depends on schema's valueType)** send data to the cloud if it's lower than this threshold
-    - `upperThreshold` **Optional (Depends on schema's valueType)** send data to the cloud if it's upper than this threshold
+    - `schema` **JSON Object** schema item, each one formed by:
+      - `typeId` **Number** semantic value type (voltage, current, temperature, etc)
+      - `valueType` **Number** data value type (boolean, integer, etc)
+      - `unit` **Number** sensor unit (V, A, W, etc)
+      - `name` **String** sensor name
+    - `event` **JSON Object** - **Optional** event item, each one formed by:
+      - `change` **Boolean** enable sending sensor data when its value changes
+      - `timeSec` **Number** - **Optional** time interval in seconds that indicates when data must be sent to the cloud
+      - `lowerThreshold` **(Depends on schema's valueType)** - **Optional** send data to the cloud if it's lower than this threshold
+      - `upperThreshold` **(Depends on schema's valueType)** - **Optional** send data to the cloud if it's upper than this threshold
   - `error` **String** a string with detailed error message
 
+  The semantic specification that defines `valueType`, `unit` and `typeId` properties can be find [here](https://knot-devel.cesar.org.br/doc/thing/unit-type-value.html).
+
+
   Success example:
 
   ```json
   {
     "id": "fbe64efa6c7f717e",
-    "config": [{
+    "config" : [{
       "sensorId": 1,
-      "change": true,
-      "timeSec": 10,
-      "lowerThreshold": 1000,
-      "upperThreshold": 3000
+      "schema": {
+        "valueType": 0xFFF1,
+        "unit": 0,
+        "typeId": 3,
```

Comment:
I have the same question as @lcbm about the values of `valueType` and `typeId`. Which swapping is correct?

Reply:
Fixed